### PR TITLE
[Feat][Format] Add internal id column to vertex payload file

### PR DESCRIPTION
**Pull request by acezen**
## Proposed changes

as title

## Types of changes

What types of changes does your code introduce to GraphAr?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/alibaba/GraphAr/blob/main/CONTRIBUTING.rst) doc
- [ ] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by acezen** [comment]
cc/ @lixueclaire , this PR needs your review.

**Comment by lixueclaire** [comment]
> cc/ @lixueclaire , this PR needs your review.

Since this is a major change, I will carefully review it shortly afterward.

File: docs/user-guide/file-format.rst [hunk 1]
```diff
@@ -57,6 +57,8 @@ Take the "person" vertex table as an example, if the chunk size is set to be 500
    :alt: vertex physical table
 
 
+Note: For efficiently utilize the filter push-down of the payload file format like Parquet, the internal vertex id is stored in the payload file as a column. And since the internal vertex id is continuous, the payload file format can use the dictionary encoding for the internal vertex id column, which would not bring too much overhead for the storage.
```
Comment (lixueclaire) [inline docs/user-guide/file-format.rst:60]: ditto

**Review by lixueclaire — APPROVED** [review]
LGTM~ Can you help to update the documentation (like file format figures and api references) as well?

**Review by Ziy1-Tan — APPROVED** [review]
LGTM!

**Pull request merged**